src/ThreeWire.h

```diff
@@ -24,6 +24,8 @@ License along with Rtc.  If not, see
 <http://www.gnu.org/licenses/>.
 -------------------------------------------------------------------------*/
 
+#include <Arduino.h>
```

Comment:
I believe the better change is to include threewire in the DS1302 header and fix the samples to remove including threewire directly.